planning/behavior_path_planner/src/scene_module/goal_planner/goal_planner_module.cpp

```diff
@@ -900,14 +900,19 @@ void GoalPlannerModule::decideVelocity()
 
 BehaviorModuleOutput GoalPlannerModule::planWithGoalModification()
 {
+  // set current road lanes, pull over lanes, and drivable lane
+  setLanes();
+
+  // if pull over path candidates generation is not finished, use previous module output
+  if (status_.get_pull_over_path_candidates().empty()) {
```

Comment:
define function like `hasPullOverPathCandidates` and retuen `status_.get_pull_over_path_candidates().empty()` is also option based on internal discussion?

Reply:
I think it is good to create a convenience function to look at multiple statuses or to combine several processes, but I think it is redundant to just look at empty.

Reply:
@kosuke55 
Make sense!